dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -66,165 +68,9 @@ const makeWindowGuardianConfig = (
-		return false;
-	};
-
-	const contributionsServiceUrl =
-		process?.env?.SDC_URL ?? CAPI.contributionsServiceUrl;
-
-	return {
-		format: CAPI.format,
-		config: {
-			frontendAssetsFullURL: CAPI.config.frontendAssetsFullURL,
-			isDev: process.env.NODE_ENV !== 'production',
-			ajaxUrl: CAPI.config.ajaxUrl,
-			shortUrlId: CAPI.config.shortUrlId,
-			pageId: CAPI.config.pageId,
-			isPaidContent: !!CAPI.config.isPaidContent,
-			showRelatedContent: CAPI.config.showRelatedContent,
-			keywordIds: CAPI.config.keywordIds,
-			ampIframeUrl: CAPI.config.ampIframeUrl,
-
-			// switches
-			switches: CAPI.config.switches,
-			abTests: CAPI.config.abTests,
-			slotBodyEnd: CAPI.config.switches.slotBodyEnd,
-			ampPrebid: CAPI.config.switches.ampPrebid,
-			permutive: CAPI.config.switches.permutive,
-			enableSentryReporting: CAPI.config.switches.enableSentryReporting,
-			enableDiscussionSwitch: CAPI.config.switches.enableDiscussionSwitch,
-			remoteBanner: CAPI.config.switches.remoteBanner,
-			remoteHeader: CAPI.config.switches.remoteHeader,
-			puzzlesBanner: CAPI.config.switches.puzzlesBanner,
-			ausMoment2020Header: CAPI.config.switches.ausMoment2020Header,
-
-			// used by lib/ad-targeting.ts
-			isSensitive: CAPI.config.isSensitive,
-			videoDuration: CAPI.config.videoDuration || 0,
-			edition: CAPI.config.edition,
-			section: CAPI.config.section,
-			sharedAdTargeting: CAPI.config.sharedAdTargeting, // missing type definition
-			adUnit: CAPI.config.adUnit,
-			discussionApiUrl: CAPI.config.discussionApiUrl,
-			discussionD2Uid: CAPI.config.discussionD2Uid,
-			discussionApiClientHeader: CAPI.config.discussionApiClientHeader,
-			idApiUrl: CAPI.config.idApiUrl,
-
-			dcrSentryDsn: CAPI.config.dcrSentryDsn,
-
-			// used by sign in gate
-			host: CAPI.config.host,
-			idUrl: CAPI.config.idUrl,
-		},
-		editionId: CAPI.editionId,
-		editionLongForm: CAPI.editionLongForm,
-		contentType: CAPI.contentType,
-		sectionName: CAPI.sectionName,
-		shouldHideReaderRevenue: CAPI.shouldHideReaderRevenue,
-		pageType: {
-			isMinuteArticle: CAPI.pageType.isMinuteArticle,
-			isPaidContent: CAPI.pageType.isPaidContent,
-			hasShowcaseMainElement: CAPI.pageType.hasShowcaseMainElement,
-		},
-		hasRelated: CAPI.hasRelated,
-		hasStoryPackage: CAPI.hasStoryPackage,
-		shouldHideAds: CAPI.shouldHideAds,
-		isAdFreeUser: CAPI.isAdFreeUser,
-		pageId: CAPI.pageId,
-		webTitle: CAPI.webTitle,
-		tags: CAPI.tags,
-		isCommentable: CAPI.isCommentable,
-		nav: {
-			readerRevenueLinks: {
-				footer: CAPI.nav.readerRevenueLinks.footer,
-				header: CAPI.nav.readerRevenueLinks.header,
-			},
-		},
-		contributionsServiceUrl,
-		isImmersive: CAPI.isImmersive,
-		isPhotoEssay: CAPI.config.isPhotoEssay || false,
-		isSpecialReport: CAPI.isSpecialReport,
-		isLiveBlog: CAPI.config.isLiveBlog || false,
-		isLive: CAPI.config.isLive || false,
-		isPreview: CAPI.config.isPreview,
-		stage: CAPI.config.stage,
-		matchUrl: CAPI.matchUrl,
-		elementsToHydrate: CAPI.blocks
```

Comment:
Have we checked that nothing is expecting this to be present on the page?

I’m thinking mostly of @guardian/commercial-dev

Reply:
I pinged comm dev and they said they are only using `window.guardian.config`. Specifically:

```
Ashish Puliyel, Yesterday 16:58
I'm afraid of commitment. So I'm scared to say we never use it. But I've looked and not seen anywhere where we use window.guardian.app
```

Reply:
Oh, right, I didn’t see it was kept on L260.